examples_utils/benchmarks/command_utils.py

```diff
@@ -120,6 +120,33 @@ def get_benchmark_variants(benchmark_name: str, benchmark_dict: dict) -> list:
     return variants
 
 
+def remove_wandb_args(cmd: str) -> str:
+    """Remove all wandb args and their values from command strings.
+    """
+
+    new_arg_list = []
+    arg_list = cmd.split(" ")
```

Comment:
I think we should use shlex.split and shlex.join like [link](https://github.com/graphcore/examples-utils/blob/master/examples_utils/benchmarks/run_benchmarks.py#:~:text=cmd%20%3D%20shlex.split(expand_environment_variables(variant_command%2C%20env))) to ensure things are passed properly to the subprocess. Same with " ".join below

Reply:
good point, will do